src/patterns/gen-ai/aws-model-deployment-sagemaker/README_custom_sagemaker_endpoint.md

```diff
@@ -0,0 +1,193 @@
+# aws-model-deployment-sagemaker (CustomSageMakerEndpoint)
+
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@cdklabs/generative-ai-cdk-constructs`|
+
+## Table of contents
+
+- [Overview](#overview)
+- [Initializer](#initializer)
+- [Pattern Construct Props](#pattern-construct-props)
+- [Pattern Properties](#pattern-properties)
+- [Default properties](#default-properties)
+- [Troubleshooting](#troubleshooting)
+- [Architecture](#architecture)
+- [Cost](#cost)
+- [Security](#security)
+- [Supported AWS Regions](#supported-aws-regions)
+- [Quotas](#quotas)
+- [Clean up](#clean-up)
+
+## Overview
+
+This construct is provided here to simplify the deployment of foundation models on Amazon SageMaker.
+
+This construct only work when region of the stack is specified explicitly:
+```
+env: {
+    region: 'eu-west-1',
+},
+```
+
+Here is a minimal deployable pattern definition:
+
+```typescript
+
+import { Construct } from 'constructs';
+import { Stack, StackProps, Aws } from 'aws-cdk-lib';
+import { CustomSageMakerEndpoint, DeepLearningContainerImage } from '@cdklabs/generative-ai-cdk-constructs';
+
+// Need an existing bucket containing model artifacts that this construct can access
+
+new CustomSageMakerEndpoint(this, 'customModel', {
+    modelId: 'bgeinf2',
+    instanceType: emergingTech.SageMakerInstanceType.ML_INF2_XLARGE,
+    container: emergingTech.DeepLearningContainerImage.fromDeepLearningContainerImage('huggingface-pytorch-inference-neuronx', '1.13.1-transformers4.34.1-neuronx-py310-sdk2.15.0-ubuntu20.04'),
+    modelDataUrl: 's3://{Bucket}/{Key}/model.tar.gz',
+    endpointName: 'testbgebase',
+    instanceCount: 1,
+    volumeSizeInGb: 100
+  });
+```
+
+## Initializer
+
+```
+new HuggingFaceSageMakerEndpoint(scope: Construct, id: string, props: IHuggingFaceSageMakerEndpointProps)
+```
```

Comment:
shouldn't this be CustomSageMakerEndpoint in place of HuggingFaceSageMakerEndpoint ?

Reply:
done in [e8ebbf5](https://github.com/awslabs/generative-ai-cdk-constructs/pull/180/commits/e8ebbf58d89f9699465624722f45c3eb9401228f)